tests/unit/gpu_av_ray_query_positive.cpp

```diff
@@ -0,0 +1,400 @@
+/* Copyright (c) 2023-2024 The Khronos Group Inc.
+ * Copyright (c) 2023-2024 Valve Corporation
+ * Copyright (c) 2023-2024 LunarG, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "../framework/layer_validation_tests.h"
+#include "../framework/pipeline_helper.h"
+#include "../framework/descriptor_helper.h"
+#include "../framework/gpu_av_helper.h"
+#include "../framework/ray_tracing_objects.h"
+#include "../../layers/gpu_shaders/gpu_shaders_constants.h"
+
+TEST_F(PositiveGpuAV, RayQueryComputeBasic) {
```

Comment:
For all tests in this file 

```suggestion
TEST_F(PositiveGpuAVRayQuery, ComputeBasic) {
```